comps/embeddings/src/integrations/tei.py

```diff
@@ -10,13 +10,14 @@
 
 from comps import CustomLogger, OpeaComponent, OpeaComponentRegistry, ServiceType
 from comps.cores.mega.utils import get_access_token
-from comps.cores.proto.api_protocol import EmbeddingRequest, EmbeddingResponse
+from comps.cores.proto.api_protocol import EmbeddingRequest, EmbeddingResponse, EmbeddingResponseData
 
 logger = CustomLogger("opea_tei_embedding")
 logflag = os.getenv("LOGFLAG", False)
 TOKEN_URL = os.getenv("TOKEN_URL")
 CLIENTID = os.getenv("CLIENTID")
 CLIENT_SECRET = os.getenv("CLIENT_SECRET")
+MODEL_ID = os.getenv("EMBEDDING_MODEL_ID", "BAAI/bge-base-en-v1.5")
```

Comment:
Should we dynamically get the model from TEI endpoint, instead of have user config here to avoid user configuration effort?

See the TEI API at https://huggingface.github.io/text-embeddings-inference/#/Text%20Embeddings%20Inference/get_model_info